shared_kv_storage/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM ubuntu:bionic
+FROM ubuntu:bionic as base_image

```

Comment:
Change it to "FROM ubuntu:bioni as build_image"

Reply:
`FROM ubuntu:bioni**c** as build_image`